docker-compose.yml

```diff
@@ -1,5 +1,12 @@
+x-common-keys-core: &common-keys-core
+  security_opt:
+    - no-new-privileges:true
+    - apparmor=docker-default
```

Comment:
I don't think we should set default values another time.
Just make a comment if you want to emphasize the use of apparmor profiles.